src/client/settings/templates/settings-cache.html

```diff
@@ -14,7 +14,7 @@
 
         <md-input-container>
             <md-checkbox ng-model="settings.config.deleteTmpOnClose">
-                Clear Tmp Folder after closing app?
```

Comment:
translations used to be externalized, it really is best not to tamper with strings. You may add new ones (although I'm not sure why you would?), but modifying or rephrasing isn't a good idea at this point, I believe.